ansible/site.yml

```diff
@@ -9,6 +9,11 @@
         name: nginx
         state: restarted
 
+    - name: install network-manager
```

Comment:
Shouldn't this be conditional to `ansible_distribution_major_version|int >= 9`?

Reply:
it's just a handler. it is called from the `network`. in the task that calls it there is a condition

Reply:
Not sure that I follow. This installs the package 'network-manager' from `apt`. That is a fairly big change.

Reply:
The matter is that this should be done when installing the `resin-wifi-connect`, but if we are connected via wifi, then when the NetworkManager is installed the network will be reconfigured and this will cause ansible fall. so I want to make sure that this will be called the last one